city_scrapers/spiders/chi_southwest_home_equity_i.py

```diff
@@ -0,0 +1,173 @@
+    r"(?:Jan(?:uary)?|Feb(?:ruary)?|Mar(?:ch)?|Apr(?:il)"
+    r"?|May|June?|July?|Aug(?:ust)?|Sep(?:tember)"
+    r"?|Oct(?:ober)?|Nov(?:ember)?|Dec(?:ember)?)"
+)
+
+TIME_REGEX = r"(?:[1-9]|1[0-2]):[0-9]{2}\s(?:AM|PM)"
+
+
+class ChiSouthwestHomeEquityISpider(CityScrapersSpider):
+    name = "chi_southwest_home_equity_i"
+    agency = "Chicago Southwest Home Equity Commission I"
+    timezone = "America/Chicago"
+    allowed_domains = ["swhomeequity.com"]
+    start_urls = ["https://swhomeequity.com/agenda-%26-minutes"]
+    location = {
+        "name": "Southwest Home Equity Assurance office",
+        "address": "5334 W. 65th Street in Chicago, Illinois",
+    }
+
+    def parse(self, response):
+        table = response.xpath(
+            "body/div/div/div/div[2]/div[3]/div/div/section/div/div[2]"
+        )
+
+        for agenda_node in table.xpath(
+            "div[@data-ux=\"GridCell\"][contains(., 'Agenda')]"
+        ):
+            minutes_node = agenda_node.xpath(
+                "following-sibling::div[@data-ux=\"GridCell\"][contains(., 'Minutes')]"
+            )
+
+            if self._get_link(minutes_node):
+                minutes_contents = self.parse_pdf(self._get_link(minutes_node))
+            else:
+                minutes_contents = None
+
+            meeting = Meeting(
+                title=(
+                    self._parse_title(minutes_contents)
+                    if minutes_contents
+                    else "Board Meeting"
+                ),
+                description="",
+                classification=COMMISSION,
+                start=self._parse_start(agenda_node),
+                end=None,
+                all_day=False,
+                time_notes=(
+                    self._parse_time_notes(minutes_contents)
+                    if minutes_contents
+                    else None
+                ),
+                location=self.location,
+                links=self._parse_links([agenda_node, minutes_node]),
+                source=self._parse_source(response),
+            )
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+            yield meeting
+
+    def _parse_title(self, text):
+        name = re.search(r"Southwest Home Equity Assurance Program", text, flags=re.I)
+        minutes = re.search(r"MINUTES", text, flags=re.I)
+        if name and minutes:
+            title = text[name.end() : minutes.start()]
+            return title.replace("\n", "").title().strip()
+        else:
+            return "Board Meeting"
+
+    def parse_pdf(self, url):
+        response = requests.get(url)
+        pdf_text = extract_text(BytesIO(response.content))
+        return pdf_text
+
+    def _parse_start(self, node):
+        # Try to get date from title
+        try:
+            date_str = self.find_date_in_title(node)
+            date_object = datetime.strptime(date_str, "%B %d %Y")
+        except (RuntimeError, ValueError):
```

Comment:
Is there a reason for including `RuntimeError` here? `ValueError` should cover all of our exceptions